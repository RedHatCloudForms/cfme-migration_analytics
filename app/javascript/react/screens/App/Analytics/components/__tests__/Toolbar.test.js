import React from 'react';
import { shallow } from 'enzyme';
import Toolbar from '../Toolbar';

describe('Toolbar component', () => {
  test('renders correctly', () => {
    expect(shallow(<Toolbar>Children Here</Toolbar>)).toMatchSnapshot();
  });
});
