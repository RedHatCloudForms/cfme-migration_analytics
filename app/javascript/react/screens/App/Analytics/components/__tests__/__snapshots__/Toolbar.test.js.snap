// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Toolbar component renders correctly 1`] = `
<Row
  bsClass="row"
  className="toolbar-pf"
  componentClass="div"
  style={
    Object {
      "paddingTop": 0,
    }
  }
>
  <Col
    bsClass="col"
    componentClass="div"
    xs={12}
  >
    <Breadcrumb
      style={
        Object {
          "marginBottom": 8,
        }
      }
      title={false}
    >
      Children Here
    </Breadcrumb>
  </Col>
</Row>
`;
